.gitignore

```diff
@@ -6,6 +6,7 @@
 *.user
 *.userosscache
 *.sln.docstates
+SWLOR.Game.Server/SWLOR.Game.Server.csproj
```

Comment:
I don't know if it's safe to gitignore the csproj file, is it?

Reply:
I've left this change in for now - it essentially means that local changes made by other people won't end up in commits.  Obviously if you need to change the project setup for some reason you'd need to un-Ignore it for that one change.